tests/behaviour/steps/inspectcli.py

```diff
@@ -81,12 +114,26 @@ def step_impl(context):
         and context.dataset_observations_info_printable
         and context.dataset_val_counts_by_measure_unit_info_printable
     )
+    # TODO: Remove above once all the tests are updated to not match strings
```

Comment:
And again?

Reply:
Yes, this is related to the old text matching style behave tests.